dbt/models/iasworld/schema/iasworld.legdat.yml

```diff
@@ -343,7 +343,7 @@ sources:
                 description: adrno should be <= 5 characters long
           - column_length:
               name: iasworld_legdat_adrsuf_zip1_taxdist_length_lte_5
-              columns:
+              column_names:
```

Comment:
I've adjusted a few tests to standardize around the parameter names `column_name` and `column_names` where appropriate, so a few tests needed to be adjusted to reflect this change.